test-runner/src/main/scala/scala/scalanative/testinterface/adapter/TestAdapter.scala

```diff
@@ -101,6 +102,7 @@ final class TestAdapter(config: TestAdapter.Config) {
     runs -= runID
   }
 
+  @nowarn("msg=getId in class Thread is deprecated")
   private[adapter] def getRunnerForThread(): ManagedRunner = {
     val threadId = Thread.currentThread().getId
```

Comment:
We can scope the `@nowarn` more precisely. Not sure if `cat=deprecation` works for JDK deprecations, but if it does it should be robust to any changes in the message. If not the `msg=...` is good 👍 

```suggestion
  private[adapter] def getRunnerForThread(): ManagedRunner = {
    val threadId = Thread.currentThread().getId: @nowarn("cat=deprecation")
```

Reply:
I tried to add both `cat` and `msg` but it didn't work. I didn't see the deprecation again but was not sure if I had to do something other than clean because the deprecation happens on sbt load.

Reply:
Bloop is telling me this - do we have to use `msg`?
```
Invalid message filter:
unknown filter: cat:deprecation
```

Reply:
Oops:
```
[info] compiling 1 Scala source to /Users/eric/workspace/scala-native/project/target/scala-2.12/sbt-1.0/classes ...
[error] /Users/eric/workspace/scala-native/test-runner/src/main/scala/scala/scalanative/testinterface/adapter/TestAdapter.scala:106:51: Invalid message filter:
[error] unknown filter: cat:deprecation
[error]     val threadId = Thread.currentThread().getId: @nowarn("cat:deprecation")
```

Reply:
Yes, should have been `cat=deprecation`.